javascriptv3/example_code/cognito-identity-provider/README.md

```diff
@@ -0,0 +1,134 @@
+<!--Generated by WRITEME on 2023-09-25 14:11:45.021685 (UTC)-->
+# Amazon Cognito Identity Provider code examples for the SDK for JavaScript (v3)
+
+## Overview
+
+Shows how to use the AWS SDK for JavaScript (v3) to work with Amazon Cognito Identity Provider.
+
+<!--custom.overview.start-->
+<!--custom.overview.end-->
+
+*Amazon Cognito Identity Provider handles user authentication and authorization for your web and mobile apps.*
+
+## ⚠ Important
+
+* Running this code might result in charges to your AWS account. For more details, see [AWS Pricing](https://aws.amazon.com/pricing/?aws-products-pricing.sort-by=item.additionalFields.productNameLowercase&aws-products-pricing.sort-order=asc&awsf.Free%20Tier%20Type=*all&awsf.tech-category=*all) and [Free Tier](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc&awsf.Free%20Tier%20Types=*all&awsf.Free%20Tier%20Categories=*all).
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+<!--custom.important.start-->
+<!--custom.important.end-->
+
+## Code examples
+
+### Prerequisites
+
+For prerequisites, see the [README](../../README.md#Prerequisites) in the `javascriptv3` folder.
+
+
+<!--custom.prerequisites.start-->
+<!--custom.prerequisites.end-->
+
+
+### Get started
+
+* [Hello Amazon Cognito](hello.js#L8) (`ListUserPools`)
+
+### Single actions
+
+Code excerpts that show you how to call individual service functions.
+
+* [Confirm a user](actions/confirm-sign-up.js#L12) (`ConfirmSignUp`)
```

Comment:
For these files that were moved, make sure there is no metadata still pointing at the old locations.

Reply:
Made some metadata changes and running a build.